lib/batch_verification/src/wire_format/v1.rs

```diff
@@ -13,6 +13,7 @@ pub struct BatchVerificationRequestWireFormatV1 {
     pub batch_number: u64,
     pub first_block_number: u64,
     pub last_block_number: u64,
+    pub pubdata_mode: u8,
```

Comment:
Is it safe to edit this file? I guess if 2FA was not rolled out in any envs then yes, but better to make sure with the platform team. There is a chance that you need to introduce a new version of wire format.